File: 3/yys/8_1924.js
var fs = require('fs')
var input = fs.readFileSync('/dev/stdin').toString().split(' ')

input[0] = parseInt(input[0])
input[1] = parseInt(input[1])

// var month = [ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12]
var days  = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

var addDay = 0

var result = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']


for(var i = 0; i < input[0] - 1; i++){
  addDay += days[i]
}

addDay += input[1]

console.log(result[addDay % 7])

File: 3/cjy/8_1924.cpp
#include <stdio.h>

int main()
{
	int result_day = 0;
	int x, y;

	char date[7][4] = { "sun", "mon", "tue", "wed", "thu", "fri", "sat" };

	scanf("%d %d", &x, &y);

	// calc days of elapsed months
	for (int i = 1; i < x; i++)
	{
		if (i == 1 || i == 3 || i == 5 || i == 7 || i == 8 || i == 10 || i == 12)
		{
			result_day += 31;
		}
		else if (i == 4 || i == 6 || i == 9 || i == 11)
		{
			result_day += 30;
		}
		else
		{
			result_day += 28;
		}
	}

	// calc days of this month
	result_day += y;

	printf("%s", date[result_day % 7]);

	return 0;
}